LondonBritishLibrary/orient/BLorient13156.xml

```diff
@@ -171,6 +185,49 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 <locus target="#23v"/>
                                 <desc>Symbolic signs referring to Christ in the margins of <locus target="#23v"/> and on various pages of the codex.</desc>
                             </decoNote>
+                            <decoNote type="band" xml:id="d10">
+                                <locus target="#5r"/>
+                                <desc>Most bands are topped by stepped elements. They vary in appearance, but the use of colours similar to those 
```

Comment:
Again, I feel that the source really needs to be mentioned.

Is there a point in repeating the same general description for each of the headpieces? Or could the information be placed somewhere else?

Reply:
Dorothea recommended to note each decorative element individually, if possible. This is useful for those, who search for decorations by the number of decorations in one manuscript. This note refers to each of the the decorations, so I repeated this phrase various times.

Reply:
Ok, it may not look nice, but it makes sense that it can be useful for some. Thank you! :)